source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -1059,6 +1062,100 @@ def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tens
     return points_xyz
 
 
+@torch.jit.script
+def convert_perspective_depth_image_to_orthogonal_depth_image(
+    perspective_depth: torch.Tensor, intrinsics: torch.Tensor
+) -> torch.Tensor:
+    r"""Converts depth images captured by the distance_to_camera replicator
```

Comment:
This is a general math function. It shouldn't refer to any replicator documentation or anything IMO